### ISSUE TITLE
Remove all new features from flask-rest-jsonapi fork

### ISSUE THREAD

**Issue by iamareebjamal** [fossasia/open-event-server#6493]
Almost all if not all features added in the fork can be shifted from it to the app and implemented and tested correctly.

The most difficult and still incorrectly implemented feature is soft deletion. Which can be implemented more robustly in this manner - 
https://github.com/miguelgrinberg/sqlalchemy-soft-delete/blob/master/app.py

It is a pain to maintain the library and most tests are failing

**Comment by kushthedude** [comment]
From the `Fossasia's fork` ?

**Comment by iamareebjamal** [comment]
Yes

**Comment by vishakhanihore** [comment]
The new features which you are referring to are the commits made by us on fork?

**Comment by kushthedude** [comment]
The new features you referring to are the commits made by us on it?

**Comment by iamareebjamal** [comment]
Yes

**Comment by kushthedude** [comment]
@iamareebjamal We can just switch the branch for this, Currently we are using `shubh-amp master` which needs to be change to `master`<, this must be enough I think. Or do we need to use latest Flask-Rest API

**Comment by prateekj117** [comment]
@kushthedude What about the changes? We still want those changes.

**Comment by prateekj117** [comment]
@iamareebjamal Why was it decided to use our own modified version of flask-rest-jsonapi instead of making desired changes in the app from the beginning?

**Comment by iamareebjamal** [comment]
There are some changes still required in the library but most of them are now implemented in the library itself. Some features implemented in the fork resulted from less research on how to achieve the result

**Comment by prateekj117** [comment]
@iamareebjamal Ok, so I will see what features are implemented in shubh-amp master, and open relevant sub-issues. I see we are also 164 commits behind original repo of flask-rest-jsonapi, and 27 commits ahead, what should be our way to go in this case?

**Comment by iamareebjamal** [comment]
Move features one by one from the fork to the app

**Comment by kushthedude** [comment]
@prateekj117 Are you checking on it ?

**Comment by prateekj117** [comment]
@kushthedude Yup. But it's going to take a lot of time and a lot of help from @iamareebjamal. You can also work on some issues.

@iamareebjamal It would be great if you could open some issues here reflecting the changes made in our fork of flask-rest-jsonapi.

**Comment by codedsun** [comment]
@iamareebjamal Can i work on this issue, also I see no branch of ` shubh-amp master`.

**Comment by codedsun** [comment]
@iamareebjamal @prateekj117  Can you tell me the first step like what feature shall be made to the app from the fork. So that we can move forward with this. Thanks

**Comment by iamareebjamal** [comment]
decide schema method
